README.md

```diff
@@ -79,12 +79,13 @@ Forte may also be compiled by directly invoking CMake by following these instruc
   make
  ```
 
-### Setting up the `PYTHONPATH`
+#### Setting up the `PYTHONPATH`
 
 If Forte is compiled with CMake, you will need to specify `PYTHONPATH` environment variable to make sure that it can be imported in python. Assuming that Forte is installed in the folder `<fortedir>`, then `PYTHONPATH` should contain `<fortedir>`. Note that if you instead include `<fortedir>/forte` in `PYTHONPATH` you will get an error.
```

Comment:
I am wondering if we should be more specific on `<fortedir>`. For example, I am now at folder `src` and I `git clone`. I will then have `src/forte`. Then I `cd forte` and do the `cmake` and `make`. Now the `__init__.py` is actually in `src/forte/forte`. So `PYTHONPATH` should include `src/forte` but not `src`.

Reply:
Good point. I made the entire procedure more explicit about folders.